src/csharp/Microsoft.Spark.Experimental/Sql/RelationalGroupedDatasetExtensions.cs

```diff
@@ -13,6 +14,34 @@ namespace Microsoft.Spark.Sql
     /// </summary>
     public static class RelationalGroupedDatasetExtensions
     {
+        /// <summary>
+        /// Maps each group of the current DataFrame using a UDF and
+        /// returns the result as a DataFrame.
+        /// 
+        /// The user-defined function should take an FxDataFrame
```

Comment:
> should take an FxDataFrame

The readers of this doc don't know what a `FxDataFrame` is. Maybe use a `<see cref` to point to the concrete type.

Reply:
Good idea